packages/js/src/executors/release-publish/release-publish.impl.ts

```diff
@@ -72,6 +76,100 @@ export default async function runExecutor(
   const registry =
     options.registry ?? execSync(`npm config get registry`).toString().trim();
   const tag = options.tag ?? execSync(`npm config get tag`).toString().trim();
+  const currentVersion = projectPackageJson.version;
+
+  try {
+    const result = execSync(npmViewCommandSegments.join(' '), {
+      env: processEnv(true),
+      cwd: packageRoot,
+      stdio: ['ignore', 'pipe', 'pipe'],
+    });
+
+    const resultJson = JSON.parse(result.toString());
+    const distTags = resultJson['dist-tags'] || {};
+    if (distTags[tag] === currentVersion) {
+      console.warn(
+        `Skipped ${packageTxt}, as v${currentVersion} already exists in ${registry} with tag "${tag}"`
```

Comment:
```suggestion
        `Skipped ${packageTxt} because v${currentVersion} already exists in ${registry} with tag "${tag}"`
```

It's a minor thing but I just think in this context "as" is a little more ambiguous and it made me pause when scanning (and removed the comma https://www.grammarly.com/blog/comma-before-because/)